src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -529,7 +532,29 @@ public static String calcMD5(final byte[] bytes) {
      * @throws IOException if the file could not be read
      */
     public static String calculateFileMD5( final File file ) throws IOException{
-        return Utils.calcMD5(FileUtils.readFileToByteArray(file));
+        return calculatePathMD5(file.toPath());
+    }
+
+    /**
+     * Calculates the MD5 for the specified file and returns it as a String
+     *
+     * @param path file whose MD5 to calculate
+     * @return file's MD5 in String form
+     * @throws IOException if the file could not be read
+     */
+    public static String calculatePathMD5(final Path path) throws IOException{
```

Comment:
Can you add a warning to the javadoc for this method that it slurps the entire file into memory, and should not be used for large files.

Reply:
Done!